ui/ui.go

```diff
@@ -96,29 +96,52 @@ func (ui *WriterUI) PrintTable(table Table) {
 	}
 }
 
-func (ui *WriterUI) AskForText(label string) (string, error) {
+func (ui *WriterUI) AskForText(opts TextOpts) (string, error) {
 	var text string
 
-	err := interact.NewInteraction(label).Resolve(&text)
-	if err != nil {
-		return "", fmt.Errorf("Asking for text: %s", err)
+	for {
+		text = opts.Default
+		err := interact.NewInteraction(opts.Label).Resolve(&text)
+		if err != nil {
+			return "", fmt.Errorf("Asking for text: %s", err)
+		}
+
+		if opts.ValidateFunc != nil {
+			isValid, err := opts.ValidateFunc(text)
+			if isValid {
+				break
```

Comment:
instead of breaking i think we can just enter here. also if ValidateFund is not specified, how would this loop exit? to make things simple if opts.ValidateFunc == nil then we should set it to func(...) (bool, error) { return true, nil } so that you dont have to have special logic in the for loop.

Reply:
Yes as of now it was not exiting when `ValidateFunc` is not specified. Made the change to set the dummy function when `ValidateFunc` is `nil`.